fea-rs/src/util/paths.rs

```diff
@@ -9,12 +9,19 @@ use std::path::{Path, PathBuf};
 /// 'font/includes/../../font/includes/features.fea'.
 fn rebase_path(path: &Path, base: &Path) -> PathBuf {
     use std::path::Component;
+
+    let mut components: Vec<_> = base.components().collect();
+
     #[cfg(not(test))]
```

Comment:
(I'm not entirely sure why these asserts were excluded when running the tests, but I kept them like that)

Reply:
good Q, and this definitely deserved a comment. Would you like to add that as part of this PR? Something like,

"This check is disabled during testing so that we can check the behaviour with non-existent paths, which fail the `is_dir` check."